src/pages/landing/stocks/_headerSection.js

```diff
@@ -0,0 +1,396 @@
+import StocksHeaderOverlayBg from 'images/svg/stocksHeaderOverlayBg.svg'
+import StocksHeaderOverlayBelowSectionBg from 'images/svg/stocksHeaderOverlayBelowSectionBg.svg'
+import StocksHeaderBelowSection from 'images/svg/stocksHeaderBelowSection.svg'
+import stocksHeaderBelowSectionM from 'images/svg/stocksHeaderBelowSection-m.svg'
+import Show from 'components/containers/show'
+import device from 'themes/device.js'
+import { localize } from 'components/localization'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+
+const query = graphql`
+    query {
+        mainHeaderImage: file(relativePath: { eq: "stocksMainHeaderImg.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const MainWrapper = styled(Flex)`
+    background-color: var(--color-white);
+    flex-direction: column;
+    overflow: hidden;
+`
+const HeaderWrapper = styled(Flex)`
+    position: relative;
+    overflow: hidden;
+    background-color: var(--color-black-8);
+    clip-path: polygon(0 0, 100% 0, 100% 100%, 0 calc(100% - 5vw));
+    @media ${device.tablet} {
+        clip-path: polygon(0 0, 100% 0, 100% 92%, 0% 100%, 0 calc(100% - 5vw));
+    }
+`
+
+const TopHeaderWrapper = styled(Flex)`
+    position: relative;
+    height: 680px;
+    width: 100%;
+    max-width: 1200px;
+    padding: 0 4rem;
+    z-index: 2;
+
+    @media ${device.desktopL} {
+        max-width: 1600px;
+    }
+    @media ${device.laptopM} {
+        height: 680px;
+        padding: 0 80px;
+        padding: 0 12rem;
+    }
+    @media ${device.laptop} {
+        height: 580px;
+        padding: 0 60px;
+    }
+    @media ${device.tablet} {
+        flex-direction: column;
+        margin-top: 24px;
+        padding: 0 20px;
+        max-width: 58.8rem;
+    }
+    @media ${device.tabletS} {
+        height: 495px;
+        padding: 0 16px;
+        margin-top: 0;
+        max-width: 100%;
+    }
+    @media ${device.mobileL} {
+        height: 385px;
+        margin-top: 10px;
+    }
+`
+const BackgroundPattern = styled.img`
+    position: absolute;
+    top: 23%;
+    right: -5%;
+    transform: translate(-50%, -50%);
+    z-index: 1;
+
+    @media ${device.tablet} {
+        left: 0;
+        top: 0;
+        right: unset;
+        transform: unset;
+        width: 262px;
+    }
+`
+const BackgroundPatternSection = styled.img`
+    position: absolute;
+    top: -3%;
+    right: -10%;
+    z-index: 1;
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
+const TopHeaderTextDiv = styled(Flex)`
+    width: 60%;
+    align-items: center;
+    @media ${device.laptopM} {
+        width: 50%;
+    }
+    @media ${device.tablet} {
+        width: 100%;
+        max-width: 460px;
+        z-index: 3;
+    }
+    @media ${device.tabletS} {
+        max-width: 100%;
+        margin-top: 10%;
+    }
+`
+const TopTextWrapper = styled(Flex)`
+    width: 100%;
+    flex-direction: column;
+    justify-content: center;
```

Comment:
Can be omitted as it defaults to center.

Reply:
Omitted in the latest commit..